cfme/tests/infrastructure/test_host.py

```diff
@@ -376,21 +376,23 @@ def test_infrastructure_hosts_refresh_multi(appliance, setup_provider_min_hosts,
                banner where "X" is the number of selected hosts. Properties for each host are
                refreshed. Making changes to test pre-commithooks
     """
+    refresh = 2
```

Comment:
_Optional_ since this is an integer I recommend calling this `nrefresh`

Reply:
updated to num_refresh to be consistent.